app/src/main/java/au/com/shiftyjelly/pocketcasts/ui/MainActivity.kt

```diff
@@ -1359,16 +1367,19 @@ class MainActivity :
                     openImport()
                 }
                 is StaffPicksDeepLink -> {
-                    val podcastListFragment = supportFragmentManager.fragments.find { it is PodcastListFragment } as? PodcastListFragment
+                    val podcastListFragment = supportFragmentManager.fragments.find { it is PodcastGridListFragment } as? PodcastGridListFragment
                     if (podcastListFragment?.listUuid != STAFF_PICKS_LIST_ID) {
-                        openTab(VR.id.navigation_discover)
-                        lifecycleScope.launch {
-                            val staffPicksList = discoverDeepLinkManager.getDiscoverList(STAFF_PICKS_LIST_ID, resources) ?: return@launch
-                            val fragment = PodcastListFragment.newInstance(staffPicksList)
-                            addFragment(fragment)
-                        }
+                        openDiscoverListDeeplink(STAFF_PICKS_LIST_ID)
+                    }
+                }
+                is TrendingDeepLink -> {
+                    val podcastListFragment = supportFragmentManager.fragments.find { it is PodcastGridListFragment } as? PodcastGridListFragment
+                    if (podcastListFragment?.inferredId != TRENDING) {
+                        openDiscoverListDeeplink(TRENDING)
                     }
                 }
+                is RecommendationsDeepLink -> {
+                }
```

Comment:
I think this will be similar to trending but the list id is `recommendations_user` if it's going to point to the "You Might Like" list. We will need to pass in the user uuid. I'm not sure if this is happening already but we shouldn't send the recommendations notification if they aren't signed in.

Reply:
Thanks for explain this. I will create a thread with Sergio and Dom because I did not talk about user being signed in or not for this notification